src/auth-service/models/Subscription.js

```diff
@@ -19,32 +19,20 @@ const SubscriptionSchema = new mongoose.Schema(
       required: true,
       unique: true,
     },
-    subscribed: {
-      type: Boolean,
-      default: true,
-    },
-    isSystemUser: {
-      type: Boolean,
-      required: true,
-    },
     notifications: {
-      twitter: {
-        type: Boolean,
-        default: true,
-      },
-      email: {
-        type: Boolean,
-        default: true,
+      mobile: {
```

Comment:
Hi @BenjaminSsempala , just curious, could there be any challenges with completely letting go of the explicit product categorisations? If so, I would like to appreciate. My worry at the moment is that we might struggle to achieve the true one account experience if we continue with the mental model of product based categorisations.

Reply:
Hello @Baalmart I was looking at it from an angle where the product defines the product the account is being used on, which would allow us or users to have more granular control of their notification preferences. 
I think as we add more notifications, we can adjust the structure accordingly to something that better categorizes the notifications we have.
Example
![image](https://github.com/airqo-platform/AirQo-api/assets/86492979/a4c7776d-8eb3-497f-824d-1bcda3aa1f77)